src/dotless.Benchmark/Program.cs

```diff
@@ -49,33 +59,33 @@ public static void Main(string[] args)
                 Console.Write('|');
             }
             Console.WriteLine();
-            Console.WriteLine(new string('-', 35 + 30*engines.Length));
+            Console.WriteLine(new string('-', 35 + 30 * engines.Length));
 
             foreach (var file in files)
             {
-                var size = rounds*contents[file].Length/1024d;
+                var size = rounds * contents[file].Length / 1024d;
                 Console.Write("{0} | {1,8:#,##0.00} Kb  | ", file.PadRight(18), size);
 
                 var times = new List<double>();
                 foreach (var engine in engines)
                 {
                     try
                     {
-                        var time = runTest(file, engine)/1000d;
+                        var time = runTest(file, engine) / 1000d;
                         times.Add(time);
-                        Console.Write("{0,8:#.00} s  {1,10:#,##0.00} Kb/s | ", time, size/time);
+                        Console.Write("{0,8:#.00} s  {1,10:#,##0.00} Kb/s | ", time, size / time);
                     }
                     catch
                     {
-                        Console.Write("Failied                     | ");
+                        Console.Write("Failed                     | ");
```

Comment:
Not sure, but the spaces here look like console column alignment, so maybe add a space now that you've removed a letter?

Reply:
Sorry, I missed that.